configCivcraft.yml

```diff
@@ -2836,11 +2836,11 @@ recipes:
     input:
       diamond ore:
         material: DIAMOND_ORE
-        amount: 20
+        amount: 32
     output:
       diamond:
         material: DIAMOND
-        amount: 24
+        amount: 64
```

Comment:
> without a lategame ore smelter can't compete with a player with one at all,

Of course they can't, why should a new player be able to compete evenly with an established one??

A while to pay off should not be months/a year it should be a week or two or else no one will bother to build it at all. 

2x is not that terribly large if the cost for the extractor is maintained(mistake on my part), it represents a clear advantage to the player and good reason to make the capital investment.